tests/test_docker_image_cache_manager.py

```diff
@@ -198,8 +200,12 @@ def test_ensure_docker_image_no_local_cache(slurm_system: SlurmSystem):
     assert result.message == ""
 
 
-def test_system_with_account(slurm_system: SlurmSystem):
-    slurm_system.account = "test_account"
+@pytest.mark.parametrize(
+    "account, gpus_per_node", [(None, None), ("test_account", None), (None, 8), ("test_account", 8)]
+)
+def test_system_with_account(slurm_system: SlurmSystem, account, gpus_per_node):
```

Comment:
Let's rename this function to something more generic, like `test_docker_import_with_extra_system_config`